modules/monitoring-example-remote-write-queue-configuration.adoc

```diff
@@ -3,28 +3,37 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: REFERENCE
-[id="example-remote-write-queue-configuration_{context}"]
-= Example remote write queue configuration
 
-// Set attributes to distinguish between cluster monitoring examples and user workload monitoring examples.
-ifndef::openshift-dedicated,openshift-rosa[]
+// The ultimate solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
+// tag::CPM[]
+[id="example-remote-write-queue-configuration-cpm_{context}"]
+= Example remote write queue configuration (CPM)
```

Comment:
same as https://github.com/openshift/openshift-docs/pull/84002/files#r1823715863

Reply:
https://84002--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/configuring-the-monitoring-stack#example-remote-write-queue-configuration-cpm_configuring-the-monitoring-stack
title is `Example remote write queue configuration (CPM)`, CPM is not replaced